Collor/Classes/CollectionCellDescribable.swift

```diff
@@ -18,10 +18,21 @@ public protocol CollectionCellDescribable : Identifiable {
     var identifier: String { get }
     var className: String { get }
     var selectable: Bool { get }
-    func getAdapter() -> CollectionAdapter
-    func size(_ collectionView: UICollectionView, sectionDescriptor: CollectionSectionDescribable) -> CGSize
+    var adapter: CollectionAdapter { get }
+    func size(_ bounds: CGRect, sectionDescriptor: CollectionSectionDescribable) -> CGSize
 }
 
+public extension CollectionCellDescribable {
+    func size(_ bounds: CGRect, sectionDescriptor: CollectionSectionDescribable) -> CGSize {
```

Comment:
`collectionViewBounds` instead of `bounds`

Reply:
Here, only vertical layout is handled. What happens if I need an horizontal layout ?